spec/rubocop/server/rubocop_server_spec.rb

```diff
@@ -7,11 +7,6 @@
 
   include_context 'cli spec behavior'
 
-  before do
-    # Makes sure the project dir of rubocop server is the isolated_environment
-    create_empty_file('Gemfile')
-  end
```

Comment:
I was trying to move `create_empty_file('Gemfile')` into "isolated environment", but it breaks more than helps.